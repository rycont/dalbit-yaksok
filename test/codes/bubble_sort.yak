목록: [3, 1, 4, 1, 5, 9]
목록 보여주기
목록_길이: 6  # TODO: 목록 길이
반복 1~목록_길이-1 의 위치1 마다
    반복 위치1+1~목록_길이 의 위치2 마다
        만약 목록[위치2] < 목록[위치1] 이면
            임시: 목록[위치1]
            목록[위치1]: 목록[위치2]
            목록[위치2]: 임시
            목록 보여주기
목록 보여주기
