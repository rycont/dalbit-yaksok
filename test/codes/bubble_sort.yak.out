[3, 1, 4, 1, 5, 9]
[1, 3, 4, 1, 5, 9]
[1, 1, 4, 3, 5, 9]
[1, 1, 3, 4, 5, 9]
[1, 1, 3, 4, 5, 9]
